packages/core/content-releases/admin/src/components/AddReleaseDialog.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  Button,
+  Grid,
+  GridItem,
+  ModalBody,
+  ModalFooter,
+  ModalLayout,
+  ModalHeader,
+  TextInput,
+  Typography,
+} from '@strapi/design-system';
+import { useNotification } from '@strapi/helper-plugin';
+import { Formik, Form } from 'formik';
+import { useIntl } from 'react-intl';
+import * as yup from 'yup';
+
+const releaseSchema = yup.object({
+  name: yup.string().required(),
+});
+
+const initialValues = {
+  name: '',
+};
```

Comment:
stylistic choice, may people prefer to write constants declared outside react components `LIKE_THIS`.